homeassistant/components/sensor/ambient_station.py

```diff
@@ -4,87 +4,80 @@
 
+_LOGGER = logging.getLogger(__name__)
+
 CONF_APP_KEY = 'app_key'
+CONF_UNITS = 'units'
 
 SENSOR_NAME = 0
 SENSOR_UNITS = 1
 
-CONF_UNITS = 'units'
 UNITS_US = 'us'
 UNITS_SI = 'si'
 UNIT_SYSTEM = {UNITS_US: 0, UNITS_SI: 1}
 
-SCAN_INTERVAL = timedelta(seconds=300)
+DEFAULT_SCAN_INTERVAL = timedelta(minutes=5)
 
 SENSOR_TYPES = {
-    'winddir': ['Wind Dir', 'º'],
-    'windspeedmph': ['Wind Speed', 'mph'],
-    'windgustmph': ['Wind Gust', 'mph'],
-    'maxdailygust': ['Max Gust', 'mph'],
-    'windgustdir': ['Gust Dir', 'º'],
-    'windspdmph_avg2m': ['Wind Avg 2m', 'mph'],
-    'winddir_avg2m': ['Wind Dir Avg 2m', 'mph'],
-    'windspdmph_avg10m': ['Wind Avg 10m', 'mph'],
-    'winddir_avg10m': ['Wind Dir Avg 10m', 'º'],
+    '24hourrainin': ['24 Hr Rain', 'in'],
+    'baromabsin': ['Abs Pressure', 'inHg'],
+    'baromrelin': ['Rel Pressure', 'inHg'],
+    'battout': ['Battery', ''],
+    'co2': ['co2', 'ppm'],
+    'dailyrainin': ['Daily Rain', 'in'],
+    'dewPoint': ['Dew Point', ['ºF', 'ºC']],
+    'eventrainin': ['Event Rain', 'in'],
+    'feelsLike': ['Feels Like', ['ºF', 'ºC']],
+    'hourlyrainin': ['Hourly Rain Rate', 'in/hr'],
     'humidity': ['Humidity', '%'],
     'humidityin': ['Humidity In', '%'],
+    'lastRain': ['Last Rain', ''],
+    'maxdailygust': ['Max Gust', 'mph'],
+    'monthlyrainin': ['Monthly Rain', 'in'],
+    'solarradiation': ['Solar Rad', 'W/m^2'],
     'tempf': ['Temp', ['ºF', 'ºC']],
     'tempinf': ['Inside Temp', ['ºF', 'ºC']],
-    'battout': ['Battery', ''],
-    'hourlyrainin': ['Hourly Rain Rate', 'in/hr'],
-    'dailyrainin': ['Daily Rain', 'in'],
-    '24hourrainin': ['24 Hr Rain', 'in'],
-    'weeklyrainin': ['Weekly Rain', 'in'],
-    'monthlyrainin': ['Monthly Rain', 'in'],
-    'yearlyrainin': ['Yearly Rain', 'in'],
-    'eventrainin': ['Event Rain', 'in'],
     'totalrainin': ['Lifetime Rain', 'in'],
-    'baromrelin': ['Rel Pressure', 'inHg'],
-    'baromabsin': ['Abs Pressure', 'inHg'],
     'uv': ['uv', 'Index'],
-    'solarradiation': ['Solar Rad', 'W/m^2'],
-    'co2': ['co2', 'ppm'],
-    'lastRain': ['Last Rain', ''],
-    'dewPoint': ['Dew Point', ['ºF', 'ºC']],
-    'feelsLike': ['Feels Like', ['ºF', 'ºC']],
+    'weeklyrainin': ['Weekly Rain', 'in'],
+    'winddir': ['Wind Dir', 'º'],
+    'winddir_avg10m': ['Wind Dir Avg 10m', 'º'],
+    'winddir_avg2m': ['Wind Dir Avg 2m', 'mph'],
+    'windgustdir': ['Gust Dir', 'º'],
+    'windgustmph': ['Wind Gust', 'mph'],
+    'windspdmph_avg10m': ['Wind Avg 10m', 'mph'],
+    'windspdmph_avg2m': ['Wind Avg 2m', 'mph'],
+    'windspeedmph': ['Wind Speed', 'mph'],
+    'yearlyrainin': ['Yearly Rain', 'in'],
 }
 
-_LOGGER = logging.getLogger(__name__)
-
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
-    vol.Required(CONF_API_KEY): cv.string,
     vol.Required(CONF_APP_KEY): cv.string,
-    vol.Required(CONF_MONITORED_CONDITIONS):
+    vol.Required(CONF_API_KEY): cv.string,
+    vol.Required(CONF_MONITORED_CONDITIONS, default=list(SENSOR_TYPES)):
```

Comment:
Shouldn't this option be optional since we have a default?

```suggestion
    vol.Optional(CONF_MONITORED_CONDITIONS, default=list(SENSOR_TYPES)):
```